lobby.py

```
@@ -10,6 +10,24 @@ def accept_match(connection: c.Connection) -> None:
+            lobbyid = 400
+        case "ranked":
+            lobbyid = 420
+        case "flex":
+            lobbyid = 440
+        case "aram":
+            lobbyid = 450
+        case "arena":
+            lobbyid = 1700
+        case _:
+            raise RuntimeError(f"invalid lobby type: {lobbytype}")
+    response = connection.api_post('lobby', {'queueId': lobbyid})
```

Ruff (0.14.0)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)